storage/rocksdb/mysql-test/rocksdb/t/rocksdb_datadir.test

```diff
@@ -21,8 +21,8 @@ let $plugin_dir=`select @@plugin_dir`;
 # Launch mysqld with non-standard rocksdb_datadir
 exec $MYSQLD_BOOTSTRAP_CMD --plugin-dir=$plugin_dir --plugin-load=$HA_ROCKSDB_SO --datadir=$ddir --rocksdb_datadir=$rdb_ddir --default-storage-engine=rocksdb --skip-innodb --default-tmp-storage-engine=MyISAM --rocksdb < $sql_file;
 
---echo Check for MANIFEST files
---list_files $rdb_ddir MANIFEST-0000*
+--echo Check for the number of MANIFEST files
+exec ls $rdb_ddir/MANIFEST-0000* | wc -l;
```

Comment:
this excludes windows - intentional?

Reply:
After updating RocksDB, the filename of the MANIFEST file may change periodically. The test is designed to check the file's number rather than its name.

Additionally, these tests with bash commands can be run using Git Bash on Windows.